sunbeam-python/sunbeam/features/shared_filesystem/feature.py

```diff
@@ -0,0 +1,111 @@
+# SPDX-FileCopyrightText: 2025 - Canonical Ltd
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+
+import click
+from packaging.version import Version
+
+from sunbeam.core.deployment import Deployment
+from sunbeam.core.manifest import (
+    CharmManifest,
+    FeatureConfig,
+    SoftwareConfig,
+)
+from sunbeam.features.interface.v1.openstack import (
+    OpenStackControlPlaneFeature,
+    TerraformPlanLocation,
+)
+from sunbeam.utils import click_option_show_hints, pass_method_obj
+from sunbeam.versions import MANILA_CHANNEL
+
+LOG = logging.getLogger(__name__)
+
+
+class SharedFilesystemFeature(OpenStackControlPlaneFeature):
+    version = Version("0.0.1")
+
+    name = "shared-filesystem"
+    tf_plan_location = TerraformPlanLocation.SUNBEAM_TERRAFORM_REPO
+
+    def default_software_overrides(self) -> SoftwareConfig:
+        """Feature software configuration."""
+        return SoftwareConfig(
+            charms={
+                "manila-k8s": CharmManifest(channel=MANILA_CHANNEL),
```

Comment:
Default to OPENSTACK_CHANNEL

Reply:
Ack, at the moment the PR was sent, we only had 2025.1 manila charms written.